### DISCUSSION TITLE
How to pause cpptools while compiling?

### DISCUSSION THREAD

**Discussion by vadi2** [microsoft/vscode-cpptools#8484] [Q&A] [answered]
I'm trying to benchmark code compilation and I see that cpptools is taking up more CPU resources than compilation itself -

![Screenshot 2021-12-03 8 30 04 AM](https://user-images.githubusercontent.com/110988/144562905-7136f719-1261-4771-8941-53cdc641eee9.png)

How can I pause the extension while I don't need it?

**Comment by sean-mcmanus** [comment, marked as answer]
You could temporarily disable the extension. Depending on what is using CPU you may be able to pause that activity, e.g. for workspace parsing or code analysis you can click the icon the status bar to make a Pause option appear. If IntelliSense is processing, you could temporarily set C_Cpp.intelliSenseEngine to "Tag Parser".
